cmd/helm-operator/main.go

```diff
@@ -103,7 +104,10 @@ func init() {
 }
 
 func main() {
-	// set glog output to stderr
+	// Ensure klog flags are initialized so they can be updated,
+	// set logging of klog to stderr so it's visible and does not
+	// log to /tmp, and finally parse our own flags.
+	klog.InitFlags(nil)
```

Comment:
are we also doing this for  the other commands?

Reply:
Not that I am aware, the only other `klog` related thing I can find in our repository is this (made by you): https://github.com/weaveworks/flux/commit/25995d57b64f7cd68c281c401bd9468dc321913c

Reply:
`klog` is just a variable name there  (i.e. it's not using the library) so it's only tangentially related.

Then, I think we should probably do it for all the commands, since `klog` is what kubernetes uses for internal error reporting, which can happen for unhandled errors. The change you mentioned makes sure we use our own logging for that in `fluxd` but I think it's still sane to initialize `klog` properly for all commands.

Reply:
Updated, also discovered logging to `stderr` is now the default.